internal/server/server.go

```diff
@@ -68,6 +68,43 @@ func FromCluster(in *redskyv1alpha1.Experiment) (redskyapi.ExperimentName, *reds
 		})
 	}
 
+	out.Constraints = nil
+	for _, c := range in.Spec.Constraints {
+		switch {
+		case c.Order != nil:
+			out.Constraints = append(out.Constraints, redskyapi.Constraint{
+				Name:           c.Name,
+				ConstraintType: redskyapi.ConstraintOrder,
+				OrderConstraint: redskyapi.OrderConstraint{
+					LowerParameter: c.Order.LowerParameter,
+					UpperParameter: c.Order.UpperParameter,
+				},
+			})
+		case c.Sum != nil:
+			sc := redskyapi.SumConstraint{
+				IsUpperBound: c.Sum.IsUpperBound,
+				Bound:        float64(c.Sum.Bound.MilliValue()) / 1000,
```

Comment:
From reading the docs and source of [`resource.Quantity`](https://github.com/kubernetes/apimachinery/blob/06deae5c9c2c030d771a467e086b6c791e8800dc/pkg/api/resource/quantity.go#L695) it seems like we should be able to do this and avoid any math and/or potential overflowing of an `int64`:

```go
Bound: float64(c.Sum.Bound.Value())
```

Same thing on line 96.

Reply:
Yeah. I'm an idiot: `x*1000/1000` is still `x`. No PhD in Maths required.

Reply:
...except when it's not.

I always forget how quantity scaling works: the scaling is applied before the rounding so `MustParse("3.5").Value() == 4` while `MustParse("3.5").MilliValue() == 3500`.

I'd like to pretend the insight to test to with `3.5` was intentional, but alas, it was dumb luck.

Reply:
Nice catch, that's a good point about the rounding.